assets/agenda/components/AgendaCoverages.tsx

```diff
@@ -61,7 +61,7 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                     >
                         <span className={classNames('coverage-item__coverage-icon', WORKFLOW_COLORS[coverage.workflow_status])}>
                             <i className={`icon-small--coverage-${getCoverageIcon(coverage.coverage_type)} me-2`}></i>
-                            <span>{`${coverage.genre.length !==0 ? gettext(coverage.genre[0].name) : getCoverageDisplayName(coverage.coverage_type)}${getSlugline(coverage)}`}</span>
+                            <span>{`${coverage.genre && coverage.genre.length !==0 ? gettext(coverage.genre[0].name) : getCoverageDisplayName(coverage.coverage_type)}${getSlugline(coverage)}`}</span>
```

Comment:
I think it might be better like `coverage.genre?.length > 0 ?` that seems more readable to me